sass/layout/_ads.scss

```diff
@@ -0,0 +1,36 @@
+@import "../base/fonts";
+
+.ads {
```

Comment:
A very minor nit-pick: consider choosing class names that are a bit more descriptive like 
ads > ads__item is alright
maybe something like ad-list > ad (even as separate blocks maybe)

auth > auth_item => auth > form_group